pom.xml

```diff
@@ -261,6 +261,11 @@
 			<version>${guava-version}</version>
 		</dependency>
 
+         <dependency>
+            <groupId>com.sap.cloud.security.xsuaa</groupId>
+            <artifactId>token-client</artifactId>
+            <version>2.10.5</version>
```

Comment:
Any special reason why you chose 2.10.5 and not [2.11.1](https://mvnrepository.com/artifact/com.sap.cloud.security.xsuaa/token-client/2.11.1)?

Reply:
> Any special reason

No special reason. I started with some example which was on that version. We now have the latest version.